CFLAGS output of pkg-config not used for libzmq

When libzmq is installed in a non-standard location (and PKG_CONFIG_PATH is set appropriately) the Makefile does not make use of ZMQ_CFLAGS and consequently the make fails with:

      CXX      zmq/libbitcoin_zmq_a-zmqabstractnotifier.o
    In file included from zmq/zmqabstractnotifier.h:8:0,
                     from zmq/zmqabstractnotifier.cpp:5:
    zmq/zmqconfig.h:16:17: fatal error: zmq.h: No such file or directory
     #include <zmq.h>
                     ^
    compilation terminated.

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -214,7 +214,7 @@ libbitcoin_server_a_SOURCES = \
 if ENABLE_ZMQ
 LIBBITCOIN_ZMQ=libbitcoin_zmq.a
 
-libbitcoin_zmq_a_CPPFLAGS = $(BITCOIN_INCLUDES)
+libbitcoin_zmq_a_CPPFLAGS = $(BITCOIN_INCLUDES) $(ZMQ_CFLAGS)
 libbitcoin_zmq_a_SOURCES = \
   zmq/zmqabstractnotifier.cpp \
   zmq/zmqnotificationinterface.cpp \

